ale/drivers/lro_drivers.py

```diff
@@ -1192,7 +1191,7 @@ def num_frames(self):
         : int
           Number of frames in the image
         """
-        return self.image_lines // (self.framelet_height // self.sampling_factor)
+        return (self.image_lines // (self.framelet_height // self.sampling_factor)) + 1
```

Comment:
Why did it need a +1?

Reply:
This was also an ISIS motivated change, the LroWideAngleCamera in ISIS calculated one extra frame. I am not sure why as the PushFrameCameraDetectorMap logic is hard to follow